### PR TITLE
Include reference on installing dependencies

### PR DESCRIPTION
**Pull request by r-chris**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,9 +25,23 @@ For debugging like setup you'll need (but since it's used for internal tools, th
...
+### Dependencies ###
+
+`argparseweb` requires `web.py` to be available. You can install it (check for the latest version) by:
+
+```bash
+wget http://webpy.org/static/web.py-0.38.tar.gz
+tar -xvzf web.py-0.38.tar.gz
+cd web.py-0.38
+python setup.py install
+```
```
Comment (nirizr) [inline README.md]: Isn't `pip install web.py` easier?

Thanks for taking the time to make usage clear!

Comment (nirizr) [reply README.md]: Also, see #6. As you pointed out, web.py is now installed as part of argparseweb.

Comment (r-chris) [reply README.md]: Yes, thanks for resolving this so quickly!

**Comment by r-chris** [comment]
Updated it as you suggested

**Comment by nirizr** [comment]
Merging, thanks!

**Pull request merged**